stubs/inertia/resources/js/Pages/Teams/TeamMemberManager.vue

```diff
@@ -351,7 +342,9 @@
             },
 
             leaveTeam() {
-                this.leaveTeamForm.delete(route('team-members.destroy', [this.team, this.$page.props.user]))
+                this.leaveTeamForm.delete(route('team-members.destroy', [this.team, this.$page.props.user]), {
```

Comment:
Missing second argument?
```suggestion
                this.leaveTeamForm.delete(route('team-members.destroy', [this.team, this.$page.props.user]), {}, {
```

Reply:
This is part of the Inertia form, which only takes options as the 2nd argument ;)

Reply:
IGNORE ME CLAUDIO 🤦